readthedocs/profiles/views.py

```diff
@@ -169,3 +176,84 @@ class TokenDeleteView(TokenMixin, DeleteView):
 
     def get_object(self, queryset=None):  # noqa
         return self.request.user.auth_token
+
+
+class UserSecurityLogView(PrivateViewMixin, ListView):
+    model = AuditLog
+    template_name = 'profiles/private/security_log.html'
+    days_limit = settings.RTD_DEFAULT_LOGS_RETENTION_DAYS
```

Comment:
This value is global for all users, don't think we can attach this value to a subscription as these are logs for a user, not for project/organization.

Reply:
This is only for the user account itself? Seems we want to query the organization that the user is part of, to determine? 

Or at least have this value be a method, so we can override it on the .com.

Reply:
If the user belongs to more than one org we could choose the max between all of them. I'm not sure if it's common to require user logs from accesses to the dashboard. We still will show logs from access to docs pages to the organization owners. But maybe it's useful to know if an account was compromised? But this info will be visible only to the user, not org owners.

Reply:
From GH:

> The security log lists all actions performed within the last 90 days.

So looks like we're good there.